gallery/visualtests/nxCollapsibleItems.js

```diff
@@ -106,6 +107,30 @@ describe('NxCollapsibleItems', function() {
     it('looks right', simpleTest(disabledTreeViewSelector));
   });
 
+  describe('NxCollapsibleItems with action content', function() {
```

Comment:
Also test focus of the collapse trigger, since we want to make sure that it does not overlap the action button

Reply:
> Also test focus of the collapse trigger, since we want to make sure that it does not overlap the action button

Good call, I added the focus test here:
aeafd77f1ecd1a29cea7b911e95387b3f1bcecac

plus the snapshots: c12e595a8a433cdf9b3b5368cf6c5001d955c4b2